.github/workflows/neofs.yml

```diff
@@ -7,13 +7,14 @@ jobs:
   push-to-neofs:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
```

Comment:
can be improved like this: https://github.com/nspcc-dev/neofs-node/blob/16e12942059fe735527a3db718143d93a117720b/.github/workflows/go.yml#L30-L31